devtools/dump_devinfo.py

```diff
@@ -207,6 +233,52 @@ async def handle_device(basedir, autosave, device: Device, batch_size: int):
         + " Do not use this flag unless you are sure you know what it means."
     ),
 )
+@click.option(
+    "--discovery-timeout",
+    envvar="KASA_DISCOVERY_TIMEOUT",
+    default=10,
+    required=False,
+    show_default=True,
+    help="Timeout for discovery.",
+)
+@click.option(
+    "-e",
+    "--encrypt-type",
+    envvar="KASA_ENCRYPT_TYPE",
+    default=None,
+    type=click.Choice(ENCRYPT_TYPES, case_sensitive=False),
+)
+@click.option(
+    "-df",
+    "--device-family",
+    envvar="KASA_DEVICE_FAMILY",
+    default="SMART.TAPOPLUG",
+    help="Device family type, e.g. `SMART.KASASWITCH`. Deprecated use `--type smart`",
```

Comment:
Should we skip adding this, if it's deprecated already?

Reply:
It's a bad comment as `dump_devinfo` doesn't  support `--type`. Removed the comment about being deprecated. In fact we should probably remove the deprecated comment from the main cli at some point as the device family will be useful going forwards.